external/fv3fit/fv3fit/train.py

```diff
@@ -105,6 +105,12 @@ def main(args):
         train_batches=train_batches,
         validation_batches=val_batches,
     )
+    if len(train_config.derived_output_variables) > 0:
+        model = DerivedModel(
+            model,
+            train_config.nonfeature_input_variables,
```

Comment:
I think this call needs to be updated to remove the `nonfeature_input_variables` attribute you made implicit.